core/src/test/scala/org/broadinstitute/dsde/rawls/workspace/WorkspaceServiceSpec.scala

```diff
@@ -2264,6 +2269,42 @@ class WorkspaceServiceSpec
     workspace.googleProjectNumber should not be empty
   }
 
+  it should "clone snapshots-by-reference" in withTestDataServices { services =>
```

Comment:
This test is mostly just testing the `MockWorkspaceManagerDAO`; since the main change here is done by WSM it's a bit indirect to test it.